### PR TITLE
model + relatiship

### PR DESCRIPTION
**Pull request by dungpt-0371**
## Related Tickets:
- [#68516](https://edu-redmine.sun-asterisk.vn/issues/68516)

## WHAT this PR does?
- ex: Tạo model và relationship

## Notes Impacted Areas:
-  app\Models

## Attachments:
image

### CODE REVIEW

**Comment by anhcnn-0378** [comment]
Oke

**Comment by dungpt-0371** [comment]
ready

File: app/Models/Cart.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class Cart extends Model
+{
+    use HasFactory;
+
+    protected $table = 'carts';
+
+    protected $fillable = [
+        'user_id',
+        'book_id',
+        'quantity'
```
Comment (ghost) [inline app/Models/Cart.php]:
```suggestion
        'quantity',
```

**Comment by dungpt-0371** [comment]
ready

**Pull request merged**